From 105462b4e76c049949b767961a944e97742845c9 Mon Sep 17 00:00:00 2001
From: "guoyin.chen" <guoyin.chen@freescale.com>
Date: Wed, 14 May 2014 09:02:14 +0800
Subject: [PATCH 307/310] ENGR00242365 Update BUILD_ID for kk4.4.2_1.0.0 GA
 rc1

Update BUILD_ID for 1.0.0_rc1
Update BUILD_NUMBER to ga release date

Signed-off-by: guoyin.chen <guoyin.chen@freescale.com>
---
 arm2_6dq/build_id.mk       | 4 ++--
 arm2_6sl/build_id.mk       | 4 ++--
 evk_6sl/build_id.mk        | 4 ++--
 hdmidongle_6dq/build_id.mk | 4 ++--
 imx5x/build_id.mk          | 4 ++--
 sabreauto_6q/build_id.mk   | 4 ++--
 sabresd_6dq/build_id.mk    | 4 ++--
 7 files changed, 14 insertions(+), 14 deletions(-)

diff --git a/arm2_6dq/build_id.mk b/arm2_6dq/build_id.mk
index 0dfe62e..2916a77 100644
--- a/arm2_6dq/build_id.mk
+++ b/arm2_6dq/build_id.mk
@@ -18,5 +18,5 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=1.0.0-beta-rc4
-export BUILD_NUMBER=20140418
+export BUILD_ID=1.0.0-rc1
+export BUILD_NUMBER=20140623
diff --git a/arm2_6sl/build_id.mk b/arm2_6sl/build_id.mk
index 0dfe62e..2916a77 100644
--- a/arm2_6sl/build_id.mk
+++ b/arm2_6sl/build_id.mk
@@ -18,5 +18,5 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=1.0.0-beta-rc4
-export BUILD_NUMBER=20140418
+export BUILD_ID=1.0.0-rc1
+export BUILD_NUMBER=20140623
diff --git a/evk_6sl/build_id.mk b/evk_6sl/build_id.mk
index 0dfe62e..2916a77 100755
--- a/evk_6sl/build_id.mk
+++ b/evk_6sl/build_id.mk
@@ -18,5 +18,5 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=1.0.0-beta-rc4
-export BUILD_NUMBER=20140418
+export BUILD_ID=1.0.0-rc1
+export BUILD_NUMBER=20140623
diff --git a/hdmidongle_6dq/build_id.mk b/hdmidongle_6dq/build_id.mk
index 0dfe62e..2916a77 100644
--- a/hdmidongle_6dq/build_id.mk
+++ b/hdmidongle_6dq/build_id.mk
@@ -18,5 +18,5 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=1.0.0-beta-rc4
-export BUILD_NUMBER=20140418
+export BUILD_ID=1.0.0-rc1
+export BUILD_NUMBER=20140623
diff --git a/imx5x/build_id.mk b/imx5x/build_id.mk
index 0dfe62e..2916a77 100644
--- a/imx5x/build_id.mk
+++ b/imx5x/build_id.mk
@@ -18,5 +18,5 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=1.0.0-beta-rc4
-export BUILD_NUMBER=20140418
+export BUILD_ID=1.0.0-rc1
+export BUILD_NUMBER=20140623
diff --git a/sabreauto_6q/build_id.mk b/sabreauto_6q/build_id.mk
index 0dfe62e..2916a77 100644
--- a/sabreauto_6q/build_id.mk
+++ b/sabreauto_6q/build_id.mk
@@ -18,5 +18,5 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=1.0.0-beta-rc4
-export BUILD_NUMBER=20140418
+export BUILD_ID=1.0.0-rc1
+export BUILD_NUMBER=20140623
diff --git a/sabresd_6dq/build_id.mk b/sabresd_6dq/build_id.mk
index 0dfe62e..2916a77 100644
--- a/sabresd_6dq/build_id.mk
+++ b/sabresd_6dq/build_id.mk
@@ -18,5 +18,5 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=1.0.0-beta-rc4
-export BUILD_NUMBER=20140418
+export BUILD_ID=1.0.0-rc1
+export BUILD_NUMBER=20140623
-- 
1.8.0

